lms/djangoapps/courseware/tests/test_group_access.py

```diff
@@ -53,7 +55,7 @@ def _wrapper(self, *args):  # pylint: disable=missing-docstring
 
 @attr('shard_2')
 @ddt.ddt
-class GroupAccessTestCase(ModuleStoreTestCase):
+class GroupAccessTestCase(ModuleStoreTestCase, MilestonesTestCaseMixin):
```

Comment:
@nasthagiri right now access.py is calling the milestones api directly, so I'll go update it to use the helpers, which I was previously unaware of. I talked with Cliff about this last week, that it seemed wrong to have to include this mixin with so many tests unrelated to milestones.